### ISSUE TITLE
Invalid Email with emailFrom()

### ISSUE THREAD

**Issue by incrediblezayed** [easazade/fake-it#2]
Hello, 
Thank You for such great package,
But I'm facing an issue right now that is:
When I generate the email with emailFrom() method I get instance of 'stringorformat'@yahoo.com,
Code: 
```
   var fakerX = FakerX.defaultInstance;
   emailController.text = fakerX.internet.emailFrom(
                    firstName: nameController.text,
                  );
```

Please respond asap

**Comment by easazade** [comment]
sorry for the late response I'm gonna look into it.

**Comment by easazade** [comment]
@incrediblezayed  Fixed in the latest release v0.6.14
Thank you for reporting this issue
